pallene/coder.lua

```diff
@@ -670,56 +661,24 @@ function Coder:init_upvalues()
     -- Functions
 
     local closures = {}
```

Comment:
Do we still need this `closures` list? My impression is that now it is equal to the `self.module.exported_functions`.